content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+---
+title: Glossary
+meta_description: "A glossary of common terms used in Ably products and the Ably documentation."
+meta_keywords: "glossary, terms, descriptions"
+section: root
+index: 50
+jump_to:
+---
+
+h2(#ably-hub). Ably Hub
+
+The Ably Hub is a service that supports the listing and discovery of realtime data streams.
+
+See the "Ably Hub documentation":/api-streamer/hub for further information.
+
+h2(#amqp). AMQP
+
+Advanced Message Queuing Protocol (AMQP) is a protocol that can be used to retrieve messages from a queue.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/Advanced_Message_Queuing_Protocol for further information.
+
+h2(#api-key). API key
+
+API keys are used to authenticate with Ably, either directly or through Ably tokens, token requests or a JWT. Each application will have at least one API key and each key can confer different rights for access to channels and other resources within its associated application. API keys contain three parts: the public app ID, the public app key ID (which combined form the API key ID) and the API key secret. API keys should be kept secret and only shared with trusted parties.
```

Comment:
Ably uses API keys for authentication, either directly ("basic authentication":/core-features/authentication#basic-authentication) or for generating tokens ("token authentication":/core-features/authentication#token-authentication). Each application will have at least one API key and each key can confer different rights for access to channels and other resources within its associated application.

API keys contain three parts: the public app ID and public app key ID (which, combined, form the API key ID) and the API key secret. API keys should be kept secret and only shared with trusted parties.

Reply:
Really good point on "allow" vs "enable". Revisited all use of them.